core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -532,7 +533,10 @@ class SparkContext(config: SparkConf) extends Logging {
           new EventLoggingListener(_applicationId, _applicationAttemptId, _eventLogDir.get,
             _conf, _hadoopConfiguration)
         logger.start()
-        listenerBus.addListener(logger)
+        listenerBus.addProcessor(
```

Comment:
I'm having a hard time finding the declaration of this method. I can't find it in your code nor in the existing master branch. Can you link to it?